uw-frame-components/js/app-config.js

```diff
@@ -33,15 +33,11 @@ define(['angular'], function(angular) {
         .value('SEARCH', {
             'searchURL': '/web/apps/search/',
         })
-        .value('FEATURES', {
-            'enabled': false,
-            'groupFiltering': false,
-            'serviceURL': 'staticFeeds/features.json',
-        })
-        .value('NOTIFICATION', {
-            'enabled': true,
+        .value('MESSAGES', {
+            'announcementsEnabled': true,
+            'notificationsEnabled': true,
```

Comment:
Maybe we don't need enabled/disabled toggles for announcements and notifications features -- maybe whether these are enabled or disabled can be implied from whether relevant data is in the messages JSON.

(Expressed [this same thought on the angularjs side where these flags are set](https://github.com/UW-Madison-DoIT/angularjs-portal/pull/674#discussion_r132476428).)